component/alert.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="/style.css" />
+  <link rel="stylesheet" href="/component/alert.css" />
+
+  <title>PineUI | Alert</title>
+</head>
+
+<body>
+  
+    <header class="header">
+      <figure class="logo-container">
```

Comment:
Great use of Semantic HTML 👍

Reply:
-Yes, I will fix my header and side navbar. I tried it but it was creating a problem so thought of doing it at the end.
-Regarding empty lines, do I have to do it manually or prettier does it automatically?

Reply:
I'm not sure but I don't think prettier does that. As far as I know prettier only breaks the `<i>` tag inside an `<a>` tag.